linuxNew/jdk/redhat/src/main/packaging/Dockerfile

```diff
@@ -4,14 +4,14 @@ RUN dnf update -y && dnf install -y rpmdevtools rpmlint gnupg2 wget tar dpkg fin
 
 RUN wget --progress=dot:mega -O /usr/bin/gosu https://github.com/tianon/gosu/releases/download/1.12/gosu-"$(dpkg --print-architecture | awk -F- '{ print $NF }')"
 RUN wget --progress=dot:mega -O /tmp/gosu.asc https://github.com/tianon/gosu/releases/download/1.12/gosu-"$(dpkg --print-architecture | awk -F- '{ print $NF }')".asc
-RUN gpg --batch --keyserver hkps://keys.openpgp.org --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4
+RUN gpg --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4
```

Comment:
Are you sure you want to change the gpg keyserver ?
No more hkp**s** ?

I can dig to find previous chat about reliability/availabilty about the gpg key server.
Some hkp hostname are juste round robin dns in front of a farm

Reply:
Yes that old address was permanently down yes

Reply:
I'll check on hkps - good spot